src/test/libcephfs/ceph_pthread_self.cc

```diff
@@ -0,0 +1,26 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2011 New Dream Network
```

Comment:
I think you can omit the copyright header. @tchaikov ?

Reply:
yeah. please drop the boilerplate header. and i don't think it's necessary to have another .cc file for this.

Reply:
Yeah, I'm a stickler from the old age where code went into .c{c} files....
Will fix.